Shoko.Server/API/v3/Models/Shoko/File.cs

```diff
@@ -107,32 +110,30 @@ public partial class File
     [JsonConverter(typeof(IsoDateTimeConverter))]
     public DateTime Updated { get; set; }
 
-    public File() { }
-
     /// <summary>
-    /// The <see cref="File.AniDB"/>, if <see cref="DataSource.AniDB"/> is
+    /// The <see cref="File.Release"/>, if <see cref="DataSource.AniDB"/> is
     /// included in the data to add.
     /// </summary>
-    [JsonProperty("AniDB", NullValueHandling = NullValueHandling.Ignore)]
-    public AniDB _AniDB { get; set; }
+    [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+    public ReleaseInfo? Release { get; set; }
 
     /// <summary>
     /// The <see cref="MediaInfo"/>, if to-be included in the response data.
     /// </summary>
     [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
-    public MediaInfo MediaInfo { get; set; }
+    public MediaInfo? MediaInfo { get; set; }
 
-    public File(HttpContext context, SVR_VideoLocal file, bool withXRefs = false, HashSet<DataSource> includeDataFrom = null, bool includeMediaInfo = false, bool includeAbsolutePaths = false) :
-        this(RepoFactory.VideoLocalUser.GetByUserIDAndVideoLocalID(context?.GetUser()?.JMMUserID ?? 0, file.VideoLocalID), file, withXRefs, includeDataFrom, includeMediaInfo, includeAbsolutePaths)
+    public File(HttpContext context, SVR_VideoLocal file, bool withXRefs = false, bool includeReleaseInfo = false, bool includeMediaInfo = false, bool includeAbsolutePaths = false) :
+        this(RepoFactory.VideoLocalUser.GetByUserIDAndVideoLocalID(context?.GetUser()?.JMMUserID ?? 0, file.VideoLocalID), file, withXRefs, includeReleaseInfo, includeMediaInfo, includeAbsolutePaths)
     { }
 
-    public File(SVR_VideoLocal_User userRecord, SVR_VideoLocal file, bool withXRefs = false, HashSet<DataSource> includeDataFrom = null, bool includeMediaInfo = false, bool includeAbsolutePaths = false)
+    public File(SVR_VideoLocal_User? userRecord, SVR_VideoLocal file, bool withXRefs = false, bool includeReleaseInfo = false, bool includeMediaInfo = false, bool includeAbsolutePaths = false)
     {
         var mediaInfo = file.MediaInfo as IMediaInfo;
         ID = file.VideoLocalID;
         Size = file.FileSize;
         IsVariation = file.IsVariation;
-        Hashes = new() { ED2K = file.Hash, MD5 = file.MD5, CRC32 = file.CRC32, SHA1 = file.SHA1 };
+        Hashes = new(file);
```

Comment:
I would just leave this as an instance of the interface, not wrapping it with a POCO.

Reply:
The interface would need to be moved or duplicated to the API layer to have proper model names in swagger I think. Currently it's wrapping it in the hash dict model used in the APIv3, which I also reused in the release info APIv3 model.